### DISCUSSION TITLE
dokan1 "Mounting disk device." event every six seconds

### DISCUSSION THREAD

**Discussion by Panabou** [dokan-dev/dokany#1001] [Q&A] [answered]
Hello, every six seconds there is a new dokan1 "Mounting disk device." entry in my event viewer logs (Win 10, 20H2), there are already more than 37000 entries.
I tried to install the latest x64 .msi from here, but the installer tells me that my installed version is already more recent.

I found the issue #971 "dokan1" spamming event viewer (no idea what dokany is lol) #971" that seems related to my problem, is there no new release yet?

Thanks and best regards!

**Comment by Liryna** [comment, marked as answer]
Hi @Panabou ,

Could you check the version of dokany you have already installed on your computer ?
The latest version has the "Mounting disk device." log message removed. So if it is still present, it means the latest version is not the one used.

Do you have an idea which product might be using dokany ? Maybe they need to update the version.

**Reply by Panabou** [reply]
Hello @Liryna !
Thanks for your answer, I had version 1.4.1.1000 on my system which was installed by the still current version (1.5.16) of Cryptomator. 
I have now manually installed the latest release of Dokany and the problem no longer occurs. I just don't understand why I didn't found the new version a few days ago, I had downloaded an installer that told me I already had the latest version installed.
I obviously somehow failed, sorry for that and thanks again!
